README.md

```diff
@@ -15,3 +15,16 @@ This action is a fork of https://github.com/tibdex/backport, it adds support for
 
 _Note:_ multiple backport labels can be added.
 For example, if a pull request has the labels `backport staging` and `backport production` it will be backported to both branches: `staging` and `production`.
+
+
+## Customizing
+
+### inputs
+
+Following inputs can be used as `step.with` keys
+
+| Name               | Type    | Description                       |
+|--------------------|---------|-----------------------------------|
+| `add_labels`       | String  | Comma separated list of labels to add to the backport PR. |
+| `github_token`     | String  | Token for the GitHub API. Set this to a personal access token (PAT) if you want github workflows to run on the opened PR. |
```

Comment:
This is pretty good, thanks! Wondering, do you know what exact permissions the PAT needs? Is giving repository access enough?

Reply:
also thanks from the MapServer team!  (this action is excellent)  thanks again.

Reply:
Trying to figure out how to run the github actions on the backport PRs, I've stumbled into more issues... I did give the backport bot PAT repository access, and marked the user as collaborator to the project, but eventulaly got:

```
remote: Permission to geoserver/geoserver.git denied to geoserver-bot.
fatal: unable to access 'https://github.com/geoserver/geoserver.git/': The requested URL returned error: 403
```

I did give it read permissions  (thought it was enough since random users with no permissions at all can open PRs), now I'm trying with Triage permissions. Do you remember what kind of relationship was setup betwene qgis-bot and the QGIS project?

Reply:
The backport branch will be created in the base repository, so the user will need to be able to create branches there too.

Reply:
Oh ok... I'm not sure Triage will work then, says "manage PRs" but does not talk about creating branches. Maybe I need to give it "write" permissions instead.

Reply:
Confirmed, the PTA needs repo access and the user has to be added as a collaborator, with write permissions.
I've shared the secret as a "repository secret" in order to make the action yaml work as written, although I believe it would be possible to make it an environment secret too, by changing the action and make it run in that same environment.

Reply:
Thank you for the feedback. Adjusted accordingly.